sandboxes/staticanalysis/staticanalyze.go

```diff
@@ -180,6 +180,14 @@ func run() (err error) {
 		return fmt.Errorf("static analysis error: %w", err)
 	}
 
+	startHashTime := time.Now()
+	archiveHash, err := utils.SHA256Hash(archivePath)
+	if err != nil {
+		return fmt.Errorf("archive checksum calculation failed: %w", err)
```

Comment:
If hashing fails, we shouldn't fail the entire static analysis. Just log a warning with `slog.WarnContext` and record an empty hash.

Reply:
Done !!